truss/config/trt_llm.py

```diff
@@ -113,5 +106,5 @@ def requires_build(self):
 
     # TODO(Abu): Replace this with model_dump(json=True)
     # when pydantic v2 is used here
-    def to_json_dict(self):
-        return json.loads(self.json())
+    def to_json_dict(self, verbose=True):
+        return json.loads(self.json(exclude_unset=not verbose))
```

Comment:
@joostinyi changes to avoid dumping all of trt llm config to yaml here

Reply:
looks good